src/axom/slam/tests/slam_map_SubMap.cpp

```diff
@@ -50,7 +50,9 @@ static PositionType const MAX_SET_SIZE = 10;
 
 TEST(slam_map, construct_empty_subsetmap)
 {
-  SubMap<int, Map<int>> m;
+  using MapType = Map<int>;
```

Comment:
It seems very strange that this fixes the issue. Nice detective work!

Reply:
indeed!

Reply:
I wonder if it is the lack of space between the `>>` characters?
(e.g. somehow `cppcheck` was choking on its handling of that?)

Reply:
That sort of thing used to be an issue a long time ago. I thought all compilers could properly handle that at this point.